Assets/Scripts/BrushCatalog.cs

```diff
@@ -189,19 +189,22 @@ public void BeginReload()
             m_GuiBrushList.Clear();
             foreach (var brush in m_GuidToBrush.Values)
             {
-                if (brush.m_HiddenInGui)
+                // Some brushes are hardcoded as hidden
+                if (brush.m_HiddenInGui) continue;
+                // Always include if experimental mode is on
```

Comment:
does this override the zapbox manifest and add them anyways? zapbox is fairly tailored to only show the brushes that work in their passthrough right now.

Reply:
Pretty sure MergeManifests handles this correctly and always returns the correct manifest. Obviously worth checking in a build.